### PR TITLE
Webhook Prints when missing

### PR DESCRIPTION
**Pull request by LeSiiN**
- added prints so people know when missing a webhook

### CODE REVIEW

File: client/cl_mugshot.lua [hunk 1]
```diff
@@ -17,10 +17,18 @@ local MugShots = {}
...
-    exports['screenshot-basic']:requestScreenshotUpload(Config.Webhook, 'files[]', {encoding = 'jpg'}, function(data)
-        local resp = json.decode(data)
-        table.insert(MugshotArray, resp.attachments[1].url)
-    end)
+    if Config.Webhook == '' then
+        print("-----------------------------")
+        print(" ")
+        print("ADD A WEBHOOK TO: Config.Webhook")
+        print(" ")
+        print("-----------------------------")
```
Comment (xFutte) [inline client/cl_mugshot.lua]:
```suggestion
        print("A webhook is missing in: Config.Webhook")
```

File: server/main.lua [hunk 2]
```diff
@@ -1770,18 +1770,26 @@ AddEventHandler("mdt:requestOfficerData", function()
...
-		  }
-	  }
-  
-	PerformHttpRequest(Config.ClockinWebhook, function(err, text, headers) end, 'POST', json.encode({username = name, embeds = embed}), { ['Content-Type'] = 'application/json' })
+	if Config.ClockinWebhook == '' then
+		print("-----------------------------")
+        print(" ")
+        print("ADD A WEBHOOK TO: Config.ClockinWebhook")
+        print(" ")
+        print("-----------------------------")
```
Comment (xFutte) [inline server/main.lua]:
```suggestion
        print("A webhook is missing in: Config.ClockinWebhook")
```

**Comment by xFutte** [comment]
There really no reason to make that error so prominent in the console. Simply adding a single line with it is fine.

**Comment by LeSiiN** [comment]
> There really no reason to make that error so prominent in the console. Simply adding a single line with it is fine.

Are u sure, people cant even see the biggest errors in the console

**Comment by xFutte** [comment]
Yes.

**Comment by LeSiiN** [comment]
> Yes.

Okay then

**Comment by xFutte** [comment]
Feel free to add a color to the log

**Comment by LeSiiN** [comment]
> Feel free to add a color to the log

Wait in Color? How?

**Review by xFutte — APPROVED** [review]

**Review by xFutte — APPROVED** [review]

**Review by xFutte — APPROVED** [review]

**Comment by xFutte** [comment]
> > Feel free to add a color to the log
> 
> Wait in Color? How?

Ask ChatGPT

**Review by xFutte — APPROVED** [review]

**Pull request merged**